controllers/users.js

```diff
@@ -602,6 +601,31 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeUserNickname = async (req, res) => {
+  try {
+    const type = req.body.type;
+    const { user } = await userQuery.fetchUser({ userId: req.params.userId });
+
+    const { discordId, username: userName } = user;
+
+    if (type === "discord") {
+      setDiscordNickname(userName, discordId);
+    }
```

Comment:
What if this condition results in a false?
Even then we will be sending a successful response?